website/docs/reference/resource-configs/postgres-configs.md

```diff
@@ -138,14 +140,34 @@ The following examples create a materialized view:
 
 </File>
 
-
 <File name='dbt_project.yml'>
 
-```yaml 
+```yaml
 models:
   path:
     materialized: materialized_view
 ```
+
+</File>
+
+### Limitations
+
+#### Changing materialization to and from "materialized_view"
+
+Swapping an already materialized model to be a materialized view and vise versa. The workaround is the manually drop the existing materialization in the data warehouse before calling `dbt run` again.
```

Comment:
Is it worth calling out that `--full-refresh` doesn't resolve this issue either? I could see that being counterintuitive since `--full-refresh` feels to me like we're starting from a blank slate (but we're not).

Reply:
```suggestion
Swapping an already materialized model to be a materialized view and vise versa. The workaround is the manually drop the existing materialization in the data warehouse before calling `dbt run` again. Normally, re-running with the `--full-refresh` flag would resolve this, but not in this case.
```